### ISSUE TITLE
Use SSL certificate from host project

### ISSUE THREAD

**Issue by mouglou** [terraform-google-modules/terraform-google-lb-http#423]
### TL;DR

We should be able to use SSL certificate from a host project, and not having to deploy the SSL certificat in each project.
When you have many service project, it come un-manageable when it will be tome to update it.

### Terraform Resources

```markdown
The impacted ressource is this one: google_compute_target_https_proxy
https://github.com/terraform-google-modules/terraform-google-lb-http/blob/master/main.tf#L114
```

### Detailed design

```markdown
Right now, with self-signed certificate, you must have the SSL certificate in the same project as the global load-balancer.
But it's hard to manage when you have many projects. 
Can't push the private key on Terraform code to create it on all projects.
We should be able to use the one from the host project (as it's a static object), like we do for other objects, like fw rules, subnet, etc...
```

### Additional information

We try both "ssl_certificates" and "certificate_maps", and build the path with the certificate from the host project, without any success.

`Error: Error creating TargetHttpsProxy: googleapi: Error 400: Invalid value for field 'resource.sslCertificates[0]': 'projects/my-host-project/global/sslCertificates/certificate-name'. Must be a valid SslCertificate URL or a valid Certificate Manager certificate URL within the same scope and project., invalid`

Someone ask about that on a Google Cloud community forum:
https://www.googlecloudcommunity.com/gc/Infrastructure-Compute-Storage/One-certificate-for-multiple-project/m-p/635332

**Comment by imrannayer** [comment]
@mouglou error is coming from API and it seems like an intended behavior. Are you able to use cross project certificate using console or CLI?

**Comment by mouglou** [comment]
> @mouglou error is coming from API and it seems like an intended behavior. Are you able to use cross project certificate using console or CLI?

Hey @imrannayer ! 
Thanks for your feedback. So I try with the console, and was not able to use the certificate too. I'm trying right now with the gcloud CLI, but seems I will not be able to use it to.

Is the best option to create an issue on the Google provider project so ?

**Comment by imrannayer** [comment]
@mouglou if you have a TAM plz ask them to create a feature request. This is GCP limitation. Terraform provider folks will not b able to help.

**Comment by mouglou** [comment]
Ok thanks a lot @imrannayer for your help.

Yes we have. I will check with him about creating this feature request so.

Have a good day !
